client/extensions/woocommerce/state/action-list/test/reducer.js

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
One more note: If an action creator has meaningful logic in it, I believe it should be tested separately. This is especially true for thunks/promises/etc. But if an action is just a one-line return, I don't see value in testing it separately.